tests/miral/static_display_config.cpp

```diff
@@ -539,3 +540,44 @@ TEST_F(StaticDisplayConfig, given_no_custom_attributes_when_they_are_added_exist
 
     EXPECT_THAT(hdmi1.custom_attribute, ElementsAre());
 }
+
+TEST_F(StaticDisplayConfig, can_set_and_retrieve_custom_data_on_layouts)
+{
+    struct CustomData
+    {
+        std::string value;
+    };
+
+    std::istringstream stream{
+        "layouts:\n"
+        "  another:\n"
+        "    cards:\n"
+        "    - VGA-1:\n"
+        "        state: disabled\n"
```

Comment:
Referring to a previous comment - could you put a custom field under VGA-1 to confirm that not just custom toplevel layout fields work?

Reply:
That is preexisting:

https://github.com/canonical/mir/blob/880f253b4dd9ca85ea26f45ca6c1e27ddad35c26/tests/miral/static_display_config.cpp#L455-L541